### DISCUSSION TITLE
objc_msgSend panic: too many int args [recovered]

### DISCUSSION THREAD

**Discussion by Charliego3** [progrium/darwinkit#55] [Q&A]
I wrap NSLayoutConstraint like this:
```go
type NSLayoutConstraint struct {
	objc.Object
}

func NewNSLayoutConstraintWithAttr(subView objc.Object, subAttribute NSLayoutAttribute, relation NSLayoutRelation,
	toItem objc.Object, toAttribute NSLayoutAttribute, multiplier float32, constant float32) NSLayoutConstraint {
	return NSLayoutConstraint{objc.Get("NSLayoutConstraint").Alloc().
		Send("constraintWithItem:attribute:relatedBy:toItem:attribute:multiplier:constant:",
			subView, subAttribute, relation, toItem, toAttribute, multiplier, constant)}
}
```
<img width="937" alt="image" src="https://user-images.githubusercontent.com/26761592/123735321-971da900-d8d1-11eb-9c1f-1be9475678ca.png">

this is NSLayoutConstraint schema
but `NewNSLayoutConstraintWithAttr` got panic: too many int args [recovered]
How to solve it?

**Comment by progrium** [comment]
This *might* work better now if it's generated since we aren't going through the variadic package's send, which has a hardcoded limit in supported arguments. Maybe we just need to add more?

**Comment by progrium** [comment]
The new system in main should work better now that this method is generated. Please let us know!

**Reply by Charliego3** [reply]
Yes, already tried DarwinKit, it is really good and amazing.
